mmv1/products/healthcare/Dataset.yaml

```diff
@@ -66,3 +75,18 @@ properties:
       The fully qualified name of this dataset
     output: true
     ignore_read: true
+  - !ruby/object:Api::Type::NestedObject
+    name: 'encryptionSpec'
+    required: false
+    immutable: true
+    default_from_api: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'kmsKeyName'
+        description: |
+          KMS encryption key that is used to secure this dataset and its sub-resources. The key used for
+          encryption and the dataset must be in the same location. If empty, the default Google encryption
+          key will be used to secure this dataset. The format is
+          projects/{projectId}/locations/{locationId}/keyRings/{keyRingId}/cryptoKeys/{keyId}.
+    required: false
```

Comment:
Are these properties part of `encryptionSpec` or KmsKeyName? That's probably what the lint error is.

Reply:
Thanks for catching this, should be fixed now.